website/views/options/social/party/loading-new-party.jade

```diff
@@ -1,5 +1,5 @@
 .container-fluid
   .row.text-center: .col-xs-12
-    h2=env.t('loadingNewParty')
+    h2=env.t('partyLoading' + partyLoadingMessage)
```

Comment:
This is referencing a server variable (hence undefined) instead of a client variable.

```jade
h2 {{::window.env.t('partyLoading' + partyLoadingMessage)}}
```